src/applications/edu-benefits/tests/components/OptOutWizard.unit.spec.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
+
+import OptOutWizard from '../../components/OptOutWizard';
+
+describe('<OptOutWizard>', () => {
+  it('should show opt out button', () => {
+    const component = mount(
+      <OptOutWizard/>);
+
+    expect(component.find('button').length).to.equal(1);
+    expect(component.state('modalOpen')).to.be.false;
+  });
+  it('should show modal when opt out button is clicked', () => {
+    // Modal uses query selector to get focus
+    const oldQuerySelector = global.document.querySelector;
```

Comment:
The `Modal.jsx` chains calls on `document` which makes this tricky to test. Should probably be refactored to use `refs` instead of `querySelector`.